src/main/java/redis/clients/jedis/MultiKeyPipelineBase.java

```diff
@@ -563,4 +563,31 @@
     client.moduleLoad(path);
     return getResponse(BuilderFactory.STRING);
   }
+
+  @Override
+  public Response<List<Long>> bitfield(String key, String... arguments) {
+    client.bitfield(key, arguments);
+    return getResponse(BuilderFactory.LONG_LIST);
+  }
+
+  @Override
+  protected Client getClient(String key) {
+    // TODO Auto-generated method stub
+    return null;
+  }
+
+  @Override
+  protected Client getClient(byte[] key) {
+    // TODO Auto-generated method stub
+    return null;
+  }
+
+  @Override
+  public Response<List<Long>> bitfield(byte[] key, byte[]... arguments) {
```

Comment:
Please remove this since PipelineBase already implemented it, and single key operations should be there.